sklearn/linear_model/tests/test_quantile.py

```diff
@@ -0,0 +1,171 @@
+# Authors: David Dale dale.david@mail.ru
+# License: BSD 3 clause
+
+import numpy as np
+from sklearn.utils.testing import assert_almost_equal, \
+    assert_raises, assert_array_almost_equal
+from sklearn.datasets import make_regression
+from sklearn.linear_model import HuberRegressor, QuantileRegressor
+from sklearn.model_selection import cross_val_score
+
+import warnings
+
+
+def test_quantile_toy_example():
+    # test how different parameters affect a small intuitive example
+    X = [[0], [1], [1]]
+    y = [1, 2, 11]
+    # for 50% quantile w/o regularization, any slope in [1, 10] is okay
+    model = QuantileRegressor(quantile=0.5, alpha=0).fit(X, y)
+    assert_almost_equal(model.intercept_, 1, 2)
+    assert model.coef_[0] >= 1
+    assert model.coef_[0] <= 10
+
+    # if positive error costs more, the slope is maximal
+    model = QuantileRegressor(quantile=0.51, alpha=0).fit(X, y)
+    assert_almost_equal(model.intercept_, 1, 2)
+    assert_almost_equal(model.coef_[0], 10, 2)
```

Comment:
use `assert_allclose` everywhere with the desired relative tolerance, as recommended in numpy docs https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_array_almost_equal.html